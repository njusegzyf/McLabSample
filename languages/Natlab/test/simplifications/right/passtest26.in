a(1).b = 4;
a(2).b = 2;
c = foo(a(3).b);
