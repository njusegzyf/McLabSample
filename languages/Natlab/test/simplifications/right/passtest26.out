a(1).b = 4;
a(2).b = 2;
t1 = a(3);
t2 = t1.b;
c = foo(t2);
